src/i18n/de.js

```diff
@@ -38,6 +38,8 @@ export default {
         medium: "Medium",
         threepids: "3PIDs",
         address: "Adresse",
+        creation_ts: "Zeitpunkt der Erstellung",
+        consent_version: "Zugestimmte Bedingungen",
```

Comment:
Bedingungen -> Nutzungsbedingungen?
How is it translated in the Apps?

Reply:
The translation: "Du kannst keine Nachrichten senden bis du die <consentLink>unsere Geschläftsbedingungen</consentLink> gelesen und akzeptiert hast."
- Geschläftsbedingungen